docs/sources/reference/components/loki/loki.source.file.md

```diff
@@ -105,6 +109,27 @@ If no file changes are detected, the poll frequency doubles until a file change
 
 If file changes are detected, the poll frequency is reset to `min_poll_frequency`.
 
+### `file_match`
+
+The `file_match` block enables built-in file discovery directly within `loki.source.file`.
+When enabled, the component discovers files on the local filesystem using glob patterns and the [doublestar][] library.
+
+The following arguments are supported:
+
+| Name                | Type                | Description                                                 | Default | Required |
+|---------------------|---------------------|-------------------------------------------------------------|---------|----------|
+| `enabled`           | `bool`              | Enable file discovery using glob patterns.                  | `false` | no       |
+| `ignore_older_than` | `duration`          | Ignores files with modification times before this duration. | `"0s"`  | no       |
+| `sync_period`       | `duration`          | How often to sync filesystem and targets.                   | `"10s"` | no       |
```

Comment:
is that a good default? we have a chance to change it

Reply:
I think the defaults are good, this is the same defaults that are using in promtail too.